src/views/CheckoutSelectAccount.vue

```diff
@@ -0,0 +1,128 @@
+                    :loginLabel="currentLogin ? currentLogin.label : ''"
+                    :loginType="currentLogin ? currentLogin.type : 0"
+                    :show-switch-login="!!this.preselectedLoginId"/>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import {Component, Emit, Vue} from 'vue-property-decorator';
+import {AccountSelector, LoginSelector} from '@nimiq/vue-components';
+import {AddressInfo} from '../lib/AddressInfo';
+import {KeyInfo, KeyStorageType} from '../lib/KeyInfo';
+import {RequestType} from '../lib/RequestTypes';
+import {State, Mutation} from 'vuex-class';
+
+@Component({components: {AccountSelector, LoginSelector}})
+export default class Checkout extends Vue {
+    @State('keys') private keys!: KeyInfo[];
+
+    @Mutation('addKey') private addKey!: (key: KeyInfo) => any;
+
+    private page: number = 1;
+    private selectedLoginId: string|null = null;
+    private preselectedLoginId: string|null = null;
+
+    private created() {
+        console.log('Keys:', this.keys);
+        if (this.keys.length === 1) {
+            this.preselectedLoginId = this.keys[0].id;
+            this.page = 2;
+        }
+    }
+
+    private get currentLogin() {
+        const loginId = this.selectedLoginId || this.preselectedLoginId || false;
+        if (!loginId) return undefined;
+        return this.keys.find((k) => k.id === loginId);
+    }
+
+    private get currentAccounts() {
+        const login = this.currentLogin;
+        if (!login) return [];
+        return Array.from(login.addresses.values());
+    }
+
+    private loginSelected(loginId: string) {
+        this.selectedLoginId = loginId;
+        this.page = 2;
+    }
+
+    private switchLogin() {
+        // TODO Redirect to import/create just like addLogin()
+
+        this.page = 1;
+    }
+
+    @Emit()
+    private addLogin() {
+        // TODO Redirect to import/create
+
+        const id: string = Math.round(Math.pow(2, 32) * Math.random()).toString(16);
+        const map = new Map<string, AddressInfo>();
+        map.set('a', new AddressInfo(
+            'a',
+            'My Account',
+            Nimiq.Address.fromString('NQ09 VF5Y 1PKV MRM4 5LE1 55KV P6R2 GXYJ XYQF'),
+        ));
+        this.addKey(new KeyInfo(id, id, map, [], KeyStorageType.LEDGER));
+    }
+
+    @Emit()
+    private accountSelected(loginId: string, address: Nimiq.Address) {
+        const key = this.keys.find((k: KeyInfo) => k.id === loginId);
+        if (!key) return;
+
+        const addressInfo = Array.from(key.addresses.values())
+            .find((ai: AddressInfo) => ai.address.equals(address));
+        if (!addressInfo) return;
+
+        this.$store.commit('setActiveAccount', {
+            loginId: key.id,
+            accountPath: addressInfo.path,
+        });
+
+        // Return to overview
+        this.backToOverview();
+    }
+
+    private backToOverview() {
+        this.$router.push({name: RequestType.CHECKOUT});
+    }
```

Comment:
I'd say anything but the backToOverview() is generic for all flows with login/account selector, right?